### DISCUSSION TITLE
image URL site preview

### DISCUSSION THREAD

**Discussion by harrisne** [transitive-bullshit/nextjs-notion-starter-kit#498] [Q&A] [answered]
hi, i want to try changing the type of preview image that shows up when a link from my website is posted on a site like twitter. i'm a hobby coder, so i'd love to give this a shot myself, but i can't seem to find the name of what this is called on my own. here are my questions: 
- is this a part of the code that can be changed readily? 
- and if so what is it called so I can find it in my own forked branch?

this kind of image: 
![image](https://user-images.githubusercontent.com/35507343/236081357-31161618-9dbc-465d-bf22-b04d04ff0143.png)

as opposed to this (here is how the preview appears on twitter)
![image](https://user-images.githubusercontent.com/35507343/236081267-928f6612-407e-4eb8-a4f2-cdf45d3759aa.png)

TY

**Comment by transitive-bullshit** [comment, marked as answer]
first off, this image is awesome 🔥 

I think you're probably better off using a custom image URL instead of trying to customize the code.

so for each blog post / page where you want to have a custom image, you need to add a "string" or "URL" property to your main table with a column called "Social Image". Then just upload your custom social image somewhere and paste in that URL to the "Social Image" field in your table / gallery view.

If the "Social Image" property exists, then that'll override the default auto-generated image.
